src/server/ftpserver.rs

```diff
@@ -216,6 +220,21 @@ where
         self
     }
 
+    /// Enable proxy protocol.
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// use libunftp::Server;
+    ///
+    /// // Use it in a builder-like pattern:
+    /// let mut server = Server::with_root("/tmp").with_proxy_protocol();
```

Comment:
The example here is missing the parameter.

Could we document exactly what should be passed here i.e. what does proxy_port mean here?

Reply:
yes I will clarify this with better documentation.